server/routes/components.ts

```diff
@@ -50,8 +63,31 @@ export default function routes({ serviceCatalogueService, redisService }: Servic
       alerts_slack_channel: alertsSlackChannel,
       github_enforce_admins_enabled: component.github_enforce_admins_enabled,
       standardsCompliance: component.standards_compliance,
+      alerts: [] as DisplayAlert[],
     }
 
+    let alerts: DisplayAlert[] = []
+    const alertManagerUrl = process.env.ALERTMANAGER_URL
+    if (alertManagerUrl) {
```

Comment:
Also I'd not bother with this, it's in config so should always be there and you have a `try/catch` block anyway.